app/views/stafftools/impersonate.js.erb

```diff
@@ -0,0 +1 @@
+$('#js-user-search-results').html("<%= escape_javascript(render partial: 'stafftools/impersonate/user_results', locals: { users: @users }) %>");
```

Comment:
Dynamic JavaScript is kind of an antipatern. Also, this could be abused by another site. On attacker.com, if I have a script tag pointing at this URL and can trick a staff user into visiting my site, the results would get loaded on attacker.com and I could collect them.

Also, I don't see where this script is getting loaded.